test/README.md

```diff
@@ -365,3 +365,19 @@ then do the following:
 
 5. Click the ***Run*** button - the newly started Pipeline will be forced to take the
    internal branches instead of the regular ones
+
+## Internal build testing in Gitlab CI
+
+In Gitlab CI we're using rules to define which stages and jobs are ran when. For running the test
+suite on internal rhel builds we're using schedules that run the test suite on latest nightly using
+the rpms that are shipped with it.
+
+We're using internal CTS instace for detecting latest accepted nightly build. It's possible to
+manually specify which `VERSION` and `TAG` you want to find or just the whole `COMPOSE_URL` directly.
+These variables can all be changed in the pipeline schedule settings. It is advised to create
+a new schedule if you want to try something or change these values, set it as inactive and then
+just trigger it manually.
+
+   !['Create pipeline schedule'](./gitlab_ci_scheduled_pipeline.png)
+
+   !['Trigger scheduled pipeline'](./gitlab_ci_schedule_trigger.png)
```

Comment:
IMO the relevant sections for Jenkins should be removed and/or the wording & images updated to document the flow for GitLab instead. We don't want to have both of them anyway and Jenkins trigger has been removed as of today.

Reply:
I've removed Jenkins part and left only Gitlab CI.